src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -99,6 +99,13 @@
 			<method signature="System.Void .ctor()" />
 		</type>
 
+		<!-- ep-rt-mono.c: ep_rt_mono_init_finish -->
+		<!-- used in ep-rt-mono.c to intialize RuntimeEventSource -->
+		<type fullname="System.Diagnostics.Tracing.RuntimeEventSource">
```

Comment:
This could go to shared SPC descriptor, coreclr has same descriptor for it

Reply:
done